include/MySugar/DashletsDialog/DashletsDialog.php

```diff
@@ -100,12 +100,9 @@ function getDashlets($category='') {
                     // in this case, we'll strip off the path information to check for the image existing
                     // in the current theme.
 
-                    $imageName = SugarThemeRegistry::current()->getImageURL(basename($dashletMeta[$files['class']]['icon']), false);
+                    $imageName = $files['class'];
                     if ( !empty($imageName) ) {
-                        if (is_file($imageName))
-                            $icon = '<img src="' . $imageName .'" alt="" border="0" align="absmiddle" />';  //leaving alt tag blank on purpose for 508
-                        else
-                            $icon = '';
+                        $icon = $imageName;
```

Comment:
It's an "edge case" but the `$icon` can be unassigned which could cause a PHP notice/warning at line 167

Reply:
I will set this to 'default' and that way it will always have a value, plus it will show the fall back icon.

Reply:
It turns out that we need this line, otherwise the icons will not show